es-ds-styles/scss/_dropdown.scss

```diff
@@ -201,3 +201,137 @@
   padding: variables.$dropdown-item-padding-y variables.$dropdown-item-padding-x;
   color: variables.$dropdown-link-color;
 }
+
+// ==========================================
+// ES-DROPDOWN CUSTOM STYLES
+// ==========================================
+
+// Custom es-dropdown input styling
+.es-dropdown-input {
```

Comment:
Can we throw a `user-select: none;` in here so that clicking a bunch of times to open and close the dropdown only opens and closes the dropdown and doesn't highlight all the text within it?

https://github.com/user-attachments/assets/e8fc124f-c520-49cc-91da-54556e132ad0